var/spack/repos/builtin/packages/node-js/package.py

```diff
@@ -51,31 +45,17 @@ class NodeJs(Package):
     version("20.15.0", sha256="01e2c034467a324a33e778c81f2808dff13d289eaa9307d3e9b06c171e4d932d")
     version("18.12.1", sha256="ba8174dda00d5b90943f37c6a180a1d37c861d91e04a4cb38dc1c0c74981c186")
     version("16.18.1", sha256="3d24c9c3a953afee43edc44569045eda56cd45cd58b0539922d17da62736189c")
-    version(
-        "14.21.1",
-        sha256="76ba961536dc11e4dfd9b198c61ff3399e655eca959ae4b66d926f29bfcce9d3",
-        deprecated=True,
-    )
-    version(
-        "14.16.1",
-        sha256="5f5080427abddde7f22fd2ba77cd2b8a1f86253277a1eec54bc98a202728ce80",
-        deprecated=True,
-    )
-    version(
-        "14.15.1",
-        sha256="a1120472bf55aea745287693a6651e16973e1008c9d6107df350126adf9716fe",
-        deprecated=True,
-    )
-    version(
-        "14.13.0",
-        sha256="8538b2e76aa06ee0e6eb1c118426c3c5ca53b2e49d66591738eacf76e89edd61",
-        deprecated=True,
-    )
-    version(
-        "14.10.0",
-        sha256="7e0d7a1aa23697415e3588a1ca4f1c47496e6c88b9cf37c66be90353d3e4ac3e",
-        deprecated=True,
-    )
+    version("14.21.1", sha256="76ba961536dc11e4dfd9b198c61ff3399e655eca959ae4b66d926f29bfcce9d3")
+    version("14.16.1", sha256="5f5080427abddde7f22fd2ba77cd2b8a1f86253277a1eec54bc98a202728ce80")
+    version("14.15.1", sha256="a1120472bf55aea745287693a6651e16973e1008c9d6107df350126adf9716fe")
```

Comment:
same for these three versions

- https://github.com/nodejs/node/blob/v14.21.1/configure#L28
- https://github.com/nodejs/node/blob/v14.16.1/configure#L24
- https://github.com/nodejs/node/blob/v14.15.1/configure#L23